SpatialGDKEditorToolbar/Source/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -79,13 +76,81 @@ bool CheckClassNameListValidity(const TArray<UClass*>& Classes)
 }
 }// ::
 
-void GenerateSchemaFromClasses(const TArray<UClass*>& Classes, const FString& CombinedSchemaPath)
+bool CheckVariableNameValidity(TSharedPtr<FUnrealType> TypeInfo)
```

Comment:
Also, why don't we put this inside `CheckClassNameListValidity` and have the whole thing be like `ValidateIdentifierNames` or something?

Reply:
Done.